connector/MongoDBAtlasODBC.pq

```diff
@@ -53,6 +66,29 @@ GetCredentialConnectionString = () as record =>
     in
         CredentialConnectionString;
 
+// Transforms the columns that the driver specifies as SQL_UNKNOWN_TYPE from binary to text values
+shared TransformSqlUnknownColumns = (table) => 
+    let
+        TableType = Value.Type(table),
+        Schema = Table.Schema(table),                        
+        SqlBinaryTypes = Table.ToRecords(Table.SelectRows(Schema,  each [TypeName] = "Binary.Type" and [NativeTypeName] <> "binData")),

```

Comment:
I updated the checking for SQL_UKNOWN types here based on a suggestion on how to improve from @nbagnard 👍